src/test/java/com/epam/tc/hw1/DivideOperationTest.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import com.epam.tc.hw1.data.TestData;
+import org.testng.annotations.Test;
+
+public class DivideOperationTest {
+
+    @Test(dataProvider = "testData", dataProviderClass = TestData.class, groups = "second")
+    public void divTest(long a, long b, long sum, long sub, long multiply, long div) {
+        Calculator calculator = new Calculator();
```

Comment:
Looks like `Calculator` could be extracted to the `@BeforeMethod` in some base class